ts/src/base/Exchange.ts

```diff
@@ -3543,7 +3543,7 @@ export default class Exchange {
     }
 
     async watchPositionForSymbols (symbols: string[] = undefined, since: Int = undefined, limit: Int = undefined, params = {}): Promise<Position[]> {
-        return this.watchPositions (symbols, since, limit, params);
+        return await this.watchPositions (symbols, since, limit, params);
```

Comment:
@carlosmiei I understand this is how we do it and for 90% of the uses cases it would work, and I think it needs to be this way to transpile currently.. but it feels wrong to await and not return a promise for an async function and also when the defined return type is a promise. Is this the best way?